kong/plugins/datadog/handler.lua

```diff
@@ -88,16 +81,23 @@ local function log(premature, conf, message)
     return
   end
 
-  local api_name   = string_gsub(message.api.name, "%.", "_")
-  local stat_name  = {
-    request_size     = api_name .. ".request.size",
-    response_size    = api_name .. ".response.size",
-    latency          = api_name .. ".latency",
-    upstream_latency = api_name .. ".upstream_latency",
-    kong_latency     = api_name .. ".kong_latency",
-    request_count    = api_name .. ".request.count",
+  local logger, err = statsd_logger:new(conf)
```

Comment:
Any reason for this change? 
Please avoid unnecessary changes.

Reply:
@hbagdi This avoid unnecessary memory allocations below if we can return early.